Sources/PackageModel/InstalledSwiftPMConfiguration.swift

```diff
@@ -40,7 +41,56 @@ public struct InstalledSwiftPMConfiguration: Codable {
                 minor: 0,
                 patch: 0,
                 prereleaseIdentifier: "latest"
-            )
+            ),
+            swiftTestingVersionForTestTemplate: defaultSwiftTestingVersionForTestTemplate
         )
     }
+
+    private static var defaultSwiftTestingVersionForTestTemplate: Version {
```

Comment:
I'd rather not have this at all and solely use config.json, `swiftSyntaxVersionForMacroTemplate` should also be removed but we can at least not add another.

Reply:
Yeah, this was driven by a failure in Linux CI in prior iterations of this patch, where we were getting a `config.json` from an older toolchain. It seemed like a reasonable policy that we would support at least slightly-older toolchains, so the default bridges that gap. If we want to be strict about this, we'll need to track down the configuration that produced this failure.